pkg/entity/datapolicy/texts.go

```diff
@@ -59,6 +59,46 @@ data_policy:
 ...
 `)
 
+var evaluateLongDocs = LongDocs(`
+Evaluates an existing data policy by applying it to sample data provided in a csv file.
+You can use this to verify the correctness of your field transforms and filters.
```

Comment:
Verify is not true, as we cannot use all transforms.

```suggestion
You can use this to test the correctness of your field transforms and filters.
```